### PR TITLE
Add security(in round, alive, dead) in check antagonists

### PR DESCRIPTION
**Pull request by AnywayFarus**
## About The Pull Request
That PR just adds a line with info about security department. How much players join on security, how many of them are alive or dead.

## Why It's Good For The Game
Staff can analyze situation with antaghonist more faster(i hope).

## Changelog

:cl: Vishenka0704
admin: separate antagonists from living player, and add another field with security info
/:cl:

### CODE REVIEW

**Comment by AnywayFarus** [comment]
It will be good if I don't lose GBP for this.
Just a small but extremely useful thing .. and three points .. eh!

**Comment by Pumpkinoe** [comment]
this is a great idea!! will make judging the station's capabilities far easier
question - how does it look with that color next to all the other colors?

**Comment by AnywayFarus** [comment]
> this is a great idea!! will make judging the station's capabilities far easier question - how does it look with that color next to all the other colors?

image

**Comment by Pumpkinoe** [comment]
hmmm, I like it a lot, bot feels weird to me that Living Players include antagonists as a part of it but security gets its own whole thing (which is honestly better, but...)
this might be out of scope for this PR, in which case disregard, but how cool would it be to have an Antagonist Players: ingame/alive/dead to throw up against security players like that? lets admins see how many antags are actually going for it

**Comment by RaveRadbury** [comment]
> It will be good if I don't lose GBP for this. Just a small but extremely useful thing .. and three points .. eh!

Consider asking maints if you can count this as a QoL PR. Features are usually for in-game things, but you're providing admin tooling to improve DM decisions.

File: code/modules/admin/check_antagonists.dm [hunk 1]
```diff
@@ -154,6 +162,7 @@
 				other_players++
 	dat += "<BR><b><font color='blue' size='3'>Players:|[connected_players - lobby_players] ingame|[connected_players] connected|[lobby_players] lobby|</font></b>"
 	dat += "<BR><b><font color='green'>Living Players:|[living_players_connected] active|[living_players - living_players_connected] disconnected|[living_players_antagonist] antagonists|</font></b>"
+	dat += "<BR><b><font color='#860e03'>Security Players:|[security] ingame||[security-security_dead] alive|[security_dead] dead|</font></b>"
```
Comment (san7890) [inline code/modules/admin/check_antagonists.dm:165]:
```suggestion
	dat += "<BR><b><font color='#860e03'>Security Players:|[security] ingame|[security-security_dead] alive|[security_dead] dead|</font></b>"
```

Consistency with the other groups formatting based on your screenshot.

File: code/modules/admin/check_antagonists.dm [hunk 2]
```diff
@@ -153,7 +164,9 @@
 			else
 				other_players++
 	dat += "<BR><b><font color='blue' size='3'>Players:|[connected_players - lobby_players] ingame|[connected_players] connected|[lobby_players] lobby|</font></b>"
-	dat += "<BR><b><font color='green'>Living Players:|[living_players_connected] active|[living_players - living_players_connected] disconnected|[living_players_antagonist] antagonists|</font></b>"
+	dat += "<BR><b><font color='green'>Living Players:|[living_players_connected] active|[living_players - living_players_connected] disconnected|</font></b>"
+	dat += "<BR><b><font color='#e29300'>Antagonists Players:|[antagonists] ingame|[antagonists-antagonists_dead] alive|[antagonists_dead] dead|</font></b>"
+	dat += "<BR><b><font color='#860e03'>Security Players:|[security] ingame|[security-security_dead] alive|[security_dead] dead|</font></b>"
```
Comment (san7890) [inline code/modules/admin/check_antagonists.dm:169]: Do mention that you're changing the format like this in your PR body

Comment (AnywayFarus) [reply code/modules/admin/check_antagonists.dm:169]: yep

**Pull request merged**